README.md

```diff
@@ -31,6 +31,17 @@
 
 <p align="center"><a href="https://github.com/AugurProject/augur-app/releases/latest"> <img width="200" src="https://augur.net/dist/images/meta_logo.png"> </a></p>
 
+## UI development
+Run the following 4 commands, each in a separate terminal session from the root of the project.
+ 
+`yarn build -w`
+
+`yarn workspace @augurproject/tools docker:geth:pop-15`
+
+`yarn workspace @augurproject/node start`
```

Comment:
```suggestion
`yarn workspace @augurproject/node clean-start`
```

Reply:
might as well describe the 3 different docker images and how to start them.